core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java

```diff
@@ -17,23 +17,29 @@
     /**
      * The log level of the root category, which is used as the default log level for all categories.
      *
-     * In addition to the standard JDK log level JBoss Logging also adds the following:
+     * JBoss Logging supports Apache style log levels:
      *
-     * {@link org.jboss.logmanager.Level#FATAL}
-     * {@link org.jboss.logmanager.Level#ERROR}
-     * {@link org.jboss.logmanager.Level#WARN}
-     * {@link org.jboss.logmanager.Level#INFO}
-     * {@link org.jboss.logmanager.Level#DEBUG}
-     * {@link org.jboss.logmanager.Level#TRACE}
+     * * {@link org.jboss.logmanager.Level#FATAL}
```

Comment:
Is this meant to use AsciiDoc style or JavaDoc style?  If the latter, this should be a `<ul>`; if the former, then they shouldn't use `{@link}`.

Reply:
` <ul>` list are not well rendered on configuration reference section on the website.
I'll remove the `{@link}`.

Reply:
On the other hand `{@link}` are ignored by the asciidoc processing so it didn't appears on the website. Removing them will restrict API discovery on IDE ...

Reply:
This is a long standing point of contention. :-)  @gsmet do you know what we settled on?  Is the doc renderer using AsciiDoc, or do we need a separate JavaDoc and AsciiDoc section, or what?

Reply:
This is both a JavaDoc and an asciidoc, and the doc build generates the HTML for the website based on asciidoc processing.
I think that the current aproach works even if it can seems disturbing because it mixes JavaDoc directive and asciidoc one.